grid.v3/client.go

```diff
@@ -198,7 +198,7 @@ func (c *Client) getWireClient(ctx context.Context, nsReceiver string) (WireClie
 
 	cc, ok := c.clientsAndConns[address]
 	if !ok {
-		conn, err := grpc.Dial(address, grpc.WithInsecure(), grpc.WithBackoffMaxDelay(20*time.Second), grpc.WithBlock())
```

Comment:
The `WithBlock` does what it says, block forever if the server is not there, so can't have that.